src/status_im/translations/fr.cljs

```diff
@@ -92,15 +90,13 @@
    :remove-from-contacts                  "Supprimer des contacts"
    :start-conversation                    "Débuter une conversation"
    :send-transaction                      "Envoyer une transaction"
-   :share-qr                              "Partager QR"
-   :error-incorrect-name                  "Veuillez choisir un autre nom"
-   :error-incorrect-email                 "Adresse e-mail incorrecte"
+   :testnet-text                          "Vous êtes connecté au réseau de test {{testnet}}. Surtout n'envoyez jamais de vrai ETH ou SNT vers vos adresses"
+   :mainnet-text                          "Vous êtes sur le réseau principale. De vrai ETH vas être envoyé"
```

Comment:
`principale` -> `principal`
`De vrai ETH vas être envoyé` sounds incorrect

Reply:
is `Vous pouvez envoyer de vrai ETH` sounds better?